partEZ/app/Event.php

```diff
@@ -25,11 +25,15 @@ class Event extends Model
         'uid', 'name', 'date', 'stime', 'etime', 'location', 'description'
     ];
 
-    public static function  getByID($eid)
+    public static function getByID($eid)
     {
```

Comment:
There is a slight difference in the return values of these functions. Originally I tried to use getByID in place of the function I created but the way it's being used forces the query to return an array instead of a single event (which is what is returned by getByID).